app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,27 @@
  *= require_tree .
  *= require_self
  */
+
+ footer {
+   position: absolute;
+   bottom: 0;
+   left: 0;
+   width: 100%;
+   font-family: "futura-pt", Futura, "Helvetica Neue", "Helvetica", "Roboto", "Arial", sans-serif;
+   font-weight: 500;
+   letter-spacing: 0.05em;
+   text-rendering: optimizeLegibility;
```

Comment:
@neoeno Half of it was us, the other half was inspiration from the Makers website.

It will inevitably conflict with the global styling at this point, I think. As we don't have it yet.